src/Lykke.Job.PayTransactionHandler.Services/Wallets/WalletsScanService.cs

```diff
@@ -127,20 +115,20 @@ public class WalletsScanService : IScanService
                             {
                                 updateRequest = tx.ToUpdateRequest();
 
-                                await _log.WriteInfoAsync(nameof(ExecuteAsync), new
+                                _log.Info("Transaction update detected", new
                                 {
                                     Hash = tx.Id,
                                     tx.WalletAddress,
                                     Blockchain = tx.Blockchain.ToString(),
                                     tx.Amount,
                                     tx.Confirmations
-                                }.ToJson(), "Transaction update detected");
+                                });
 
                                 await _payInternalClient.UpdateTransactionAsync(updateRequest);
                             }
                             catch (Exception ex)
                             {
-                                await _log.WriteErrorAsync(nameof(ExecuteAsync), updateRequest?.ToJson(), ex);
+                                _log.Error(ex, context: updateRequest);
```

Comment:
Alexey Novicikhin usually suggests to save in context only data wich is used for searching by fields in kibana.